.travis.yml

```
@@ -8,13 +8,15 @@ python:
 install:
   - pip install requests
   - pip install tdigest
+  - pip install uri
+  - pip install Deprecated
   - pip install flake8 flake8-colors flake8-import-order
   - pip install pep8-naming
   - pip install pydocstyle
   - pip install pylint
 before_script:
   - python -m flake8
-  - python -m pylint wavefront_sdk
+  - python -m pylint -d duplicate-code wavefront_sdk
```

```suggestion
  - python -m pylint wavefront_sdk
```
We should try to not duplicate code
If it's not possible, please use inline`pylint disable=duplicate-code` directives explicitly

---

I did not change the existing code to manage the duplicate code as we have marked old clients as deprecated and we might remove in the near future.
I tried using the `pylint disable=duplicate-code` inline but did not work for me. there is an open issue for this in github
https://github.com/PyCQA/pylint/issues/214